Le_duong/LoadDB/connect.php

```diff
@@ -0,0 +1,52 @@
+<?php
+	 //Connect server 
+	$servername="localhost";
+	$username="leduong";
+	$password="Leduong.4dec";
+	$dbname="php2104";
+	$conn = new mysqli($servername, $username, $password,$dbname);
+
+	// Check connection
+	if ($conn->connect_error) 
+	{
+	  die("Connection failed: " . $conn->connect_error);
+	}
+
+
+	// Post data
+	$first_name=$last_name=$address=$email=$phone='';
```

Comment:
```suggestion
	$firstName = '';
	$lastName = '';
	$address = '';
	$email = '';
	$phone = '';
```
Viết như e ko sai, nhưng ko theo chuẩn conversion php